cmd/deploy/deploy.go

```diff
@@ -198,6 +200,17 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 
 	defer dc.cleanUp(ctx)
 
+	c, _, err := dc.k8sClientProvider.Provide(okteto.Context().Cfg)
+	if err != nil {
+		return err
+	}
+
+	cfg := deploy.TranslateConfigMap(ctx, opts.Name)
```

Comment:
This may impact on how a pipeline is displayed in the UI when the `okteto deploy` command is executed from the pipeline installer (where another config map is created). Maybe there is nothing to do as we are doing something similar with stacks but we should test it

Reply:
We will set the annotation on every resource instead of setting it on the cfgMap